rpc/vk_hash/src/lib.rs

```diff
@@ -83,6 +103,26 @@ impl VKHash {
 }
 
 impl VKHashApiServer<H256> for VKHash {
+    fn fflonk(&self, vk: FflonkVk) -> RpcResult<H256> {
+        let vk = Vk {
```

Comment:
Implement `From<FflonkVk> for pallet_fflonk_verifier::Vk` and use `into()` is nicer and idiomatic (also for groth16).

Reply:
Done